### ISSUE TITLE
RSWires incompability - Bundled Cables

### ISSUE THREAD

**Issue by hdainester** [MightyPirates/OpenComputers#2809]
BundledCables dont connect to Computers with a RedstoneCard T2 or to the Redstone I/O block so it's impossible to read bundled output by those components.

Minecraft: 1.12.2
OpenComputers: 1.7.2.67
RSWires: 3.0.0

Similiar to #2401

**Comment by lena-kze** [comment]
1.10.2 am I right?
If so, ProjectRed wasn't that dev'ed any more.

**Comment by hdainester** [comment]
Yes that's why this issue is not about ProjectRed but about RSWires, it's yet another successor to RedPower I recently stumbled accross :).

**Comment by lena-kze** [comment]
Nice! I am still searching for a bundled cable replacement mod. Would be cool if a successor can be supported by OC too.

Damn, learning Java would put me upside down. And all those forge apis. Arrg...

**Comment by payonel** [comment]
I definitely am in support of more bundled redstone interop from OC. Thanks for finding this and reporting

**Comment by LordFokas** [comment]
@snahreteid do you mean another successor to **RedPower**? :p

**Comment by hdainester** [comment]
@LordFokas Indeed, thanks for correction.

**Comment by payonel** [comment]
After some review -- I've decided to cut this from our 1.7.3 release
We do currently support bundled redstone in project red. But going forward OC should either create a bundled redstone interface, and ask rswires (et al) to use it, or use some generally accepted bundled redstone public api if one exists. I'm open to suggestions (provide them here)

**Comment by SquidDev** [comment]
I'm not aware of some "standard" bundled cable API, but it would be something I'd be interested in seeing - ComputerCraft has the same joys of trying to get every mod to play well with it. 

I wonder if this is something which could be added to Forge as a capability, like the Forge energy system. Even something as simple as asie's [`IBundledReceiver`](https://git.asie.pl/asie-minecraft/Charset/src/branch/1.12/src/main/java/pl/asie/charset/api/wires/IBundledReceiver.java) would probably be sufficient.

**Comment by asiekierka** [comment]
>to Forge

*shivers*
